app/retail/views.py

```diff
@@ -735,9 +735,13 @@ def help(request):
         'url': 'https://medium.com/gitcoin/tutorial-how-to-price-work-on-gitcoin-49bafcdd201e',
         'title': _('How to Price Work on Gitcoin'),
     }, {
-        'img': static('v2/images/help/tools.png'),
-        'url': 'https://medium.com/gitcoin/tutorial-post-a-bounty-in-90-seconds-a7d1a8353f75',
-        'title': _('Post a Bounty in 90 Seconds'),
+         'img': 'https://raw.githubusercontent.com/Anish-Agnihotri/Gitcoin-Exemplars/master/helpImage.png',
+         'url': 'https://github.com/Anish-Agnihotri/gitcoin-exemplars',
```

Comment:
we should probably move the source of truth for these from https://github.com/Anish-Agnihotri/gitcoin-exemplars to the `gitcoinco` org

same comment applies for all the other links too

Reply:
Yes, that would make sense. 

I can revise this PR by including the file structure and .md files to the `docs` folder, and referencing it from there. Or, just let me know a directory that works for you.

Reply:
sure the repo is https://github.com/gitcoinco/Gitcoin-Exemplars -- i just sent you an invite to write to it!